curated_transformers/models/attention.py

```diff
@@ -307,8 +181,7 @@ def __init__(
         hidden_width: int,
         num_attention_heads: int,
         qkv_mode: QkvMode,
-        rotary_fraction: float,
-        rotary_base: int = 10000,
+        rotary_embeds: Optional[RotaryEmbeddingConfig],
```

Comment:
```suggestion
        rotary_embeds: Optional[RotaryEmbeddingConfig] = None,
```
Do we want to default to `None` here as well?